django/db/migrations/operations/models.py

```diff
@@ -465,7 +465,10 @@ def database_backwards(self, app_label, schema_editor, from_state, to_state):
         return self.database_forwards(app_label, schema_editor, from_state, to_state)
 
     def describe(self):
-        return "Rename table for %s to %s" % (self.name, self.table)
+        return "Rename table for %s to %s" % (
+            self.name,
+            self.table if self.table is not None else "(default)"
```

Comment:
I meant `self.table or '(default)'` seems more neat and intuitive, but this way it is more explicit. :)